www/js/index.js

```diff
@@ -136,19 +128,24 @@ function findLine(e) {
     lineFound.click();
   }
 }
-
 // Keyboard shortcuts
 if (typeof Mousetrap !== 'undefined') {
   Mousetrap.bindGlobal('esc', escKey);
 
-  Mousetrap.bindGlobal(['command+1', 'ctrl+1'], waheguruSlide);
-  Mousetrap.bindGlobal(['command+2', 'ctrl+2'], moolMantraSlide);
-  Mousetrap.bindGlobal(['command+3', 'ctrl+3'], emptySlide);
-  Mousetrap.bindGlobal(['command+4', 'ctrl+4'], anandSahibBhog);
-  Mousetrap.bindGlobal(['command+5', 'ctrl+5'], helpGuideShortcut);
-  Mousetrap.bindGlobal(['command+6', 'ctrl+6'], legendShortcut);
+  Mousetrap.bindGlobal(['mod+1'], slideShortcuts.waheguru);
+  Mousetrap.bindGlobal(['mod+2'], slideShortcuts.moolMantra);
+  Mousetrap.bindGlobal(['mod+3'], slideShortcuts.empty);
 
-  Mousetrap.bindGlobal(['command+/', 'ctrl+/'], searchBarShortcut);
+  Mousetrap.bindGlobal(['mod+4'], ceremonyShortcuts.anandSahibBhog);
+
+  Mousetrap.bindGlobal('mod+5', interfaceShortcuts.help);
+  Mousetrap.bindGlobal('mod+6', interfaceShortcuts.legend);
+  Mousetrap.bindGlobal('mod+/', interfaceShortcuts.searchBar);
+  Mousetrap.bindGlobal('mod+g', () => {
```

Comment:
@saintsoldierx @maneetpaul please review if this is what we are going for in terms of UX. 
![image](https://user-images.githubusercontent.com/2982784/60608550-affc2e00-9d8d-11e9-8884-9e3963bc2b30.png)
![image](https://user-images.githubusercontent.com/2982784/60608567-bbe7f000-9d8d-11e9-97f7-c7816633dc0a.png)

Reply:
This looks great. I would like it if we can use the arrow keys to navigate through everything so you don't need to use your mouse. Also, what shows up if you open "Slides" ?

Reply:
Also, is the shortcut to open this CMD + G?

Reply:
1. not sure
2. yes